.github/workflows/base-glibc-busybox-bash.yaml

```diff
@@ -14,23 +14,47 @@ on:
 jobs:
   build:
     name: Build & Push
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
+    container:
+      # travier/podman-action contains newer podman/buildah versions.
+      image: quay.io/travier/podman-action
+      options: --privileged
```

Comment:
refs:
- https://github.com/redhat-actions/buildah-build/issues/131

Reply:
This should fix https://github.com/bioconda/bioconda-utils/issues/958 since this uses [`buildah >=1.35.0`](https://github.com/containers/buildah/releases/tag/v1.35.0) which includes https://github.com/containers/storage/pull/1847 .
On the Docker side, the issue should also be resolved for older images when `docker >=25.0.4`](https://github.com/moby/moby/releases/tag/v25.0.4) onward is used.